lib/tpm_tcg2.c

```diff
@@ -21,38 +21,39 @@
 #include "tpm-utils.h"
 #include <bloblist.h>
 
-int tcg2_get_pcr_info(struct udevice *dev, u32 *supported_pcr, u32 *active_pcr,
-		      u32 *pcr_banks)
+int tcg2_get_pcr_info(struct udevice *dev, u32 *supported_bank, u32 *active_bank,
```

Comment:
I don't think we need this apart from the PCR bank renaming. As I mentioned the tcg2 protocol selects all u-boots active algorithms. What might be problematic is finding a TPM that supports more algorithms that u-boot has, but I think we cover this already